packages/core/networks/42.json

```diff
@@ -98,5 +98,9 @@
   {
     "contractName": "RangeBondLongShortPairFinancialProductLibrary",
     "address": "0x09381C827438605B5ea3e760A27666Be3497602C"
+  },
+  {
+    "contractName": "BridgeAdmin",
```

Comment:
Why not add BridgePool to this file?

Reply:
there is more than 1 `BridgePool` per network! how would we add `n` number of them to this file?

Reply:
I guess not since we'll never just call `getAddress("BridgePool")` since the caller will need to know which one they want!